contracts/StorageAccessible.sol

```diff
@@ -63,10 +63,11 @@ contract StorageAccessible {
     function simulateDelegatecallInternal(
         address targetContract,
         bytes memory calldataPayload
-    ) public returns (bytes memory) {
+    ) public returns (bytes memory res) {
         (bool success, bytes memory response) = targetContract.delegatecall(
             calldataPayload
         );
+        res = response;
```

Comment:
Note that here I have tried to use response directly as the return var name and then remove the `bytes memory` from where response is originally declared. However, this did not work as above because some identifier is expected....

<img width="545" alt="Screenshot 2020-12-02 at 11 27 58" src="https://user-images.githubusercontent.com/11778116/100860937-71423a00-3491-11eb-975e-ef2827756733.png">

Reply:
Somewhat weird error (sounds like a compiler bug we could raise to the solidity team for help). 

In the meantime, does this work?

```solidity
    function simulateDelegatecallInternal(
        address targetContract,
        bytes memory calldataPayload
    ) public returns (bytes memory response) {
        bool success;
        (success, response) = targetContract.delegatecall(calldataPayload);
        revertWith(abi.encodePacked(response, success));
    }
```

Reply:
Good thinking, I will try this!

Reply:
YES!  This is much much better!

Reply:
I have made the change and also reported this in https://gitter.im/ethereum/solidity

Reply:
Response makes sense "it is unfortunate, but you cannot mix variable declarations with assignments to existing variables"